<ui version="4.0" >
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Загрузка организаций ЛПУ из ЕИС.ОМС</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>импортировать из</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSplitter" name="splitter" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QLineEdit" name="edtFileName" >
        <property name="text" >
         <string/>
        </property>
        <property name="readOnly" >
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="edtIP" >
        <property name="readOnly" >
         <bool>false</bool>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2" >
    <widget class="CProgressBar" name="progressBar" >
     <property name="value" >
      <number>24</number>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="QLabel" name="statusLabel" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2" >
    <widget class="QTextBrowser" name="log" />
   </item>
   <item row="5" column="0" colspan="2" >
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnImport" >
       <property name="text" >
        <string>начать импортирование</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelNum" >
       <property name="text" >
        <string>всего записей в источнике:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose" >
       <property name="text" >
        <string>закрыть</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CProgressBar</class>
   <extends>QProgressBar</extends>
   <header location="global" >library.ProgressBar</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
